code/game/objects/items/melee/misc.dm

```diff
@@ -290,9 +291,15 @@
 			if (stun_animation)
 				user.do_attack_animation(target)
 
-			playsound(get_turf(src), on_stun_sound, 75, TRUE, -1)
```

Comment:
You deleted the sound here and never readded it.

Reply:
also I don't hear a sound for contractor batons but im not sure if thats intentional or not

Reply:
Oh whoops thanks for that catch.